packages/circuits/jwt-verifier-test.circom

```diff
@@ -0,0 +1,5 @@
+pragma circom 2.1.6;
+
+include "./jwt-verifier.circom";
+
+component main  = JWTVerifier(121, 17, 1024, 128, 896, 72, 605);
```

Comment:
@shreyas-londhe 
I think this file is not necessary if you already have the test circuit in tests/test-circuits.

Reply:
@SoraSuegami This was part of the rebase, I'll delete it